public/components/metrics/sidebar/sidebar.tsx

```diff
@@ -43,10 +43,12 @@ export const Sidebar = ({
 
   useEffect(() => {
     if (additionalMetric) {
```

Comment:
what's an additional metric?

Reply:
A badly named variable.

It's used when navigating to a specific metric for "edit", the URL is ./metrics/{savedObjectId}.

I tried a bunch fo names that didn't work and then just gave up trying.  (Not the best approach)